extensions/amp-viewer-integration/0.1/messaging/package.json

```diff
@@ -1,11 +1,15 @@
 {
-  "name": "amp-viewer-messaging",
-  "version": "1.0.2",
+  "name": "@ampproject/viewer-messaging",
+  "version": "1.1.0",
   "description": "Messaging between an AMP Doc and a Viewer",
   "author": "The AMP HTML Authors",
   "license": "Apache-2.0",
+  "main": "messaging.js",
+  "types": "messaging.d.ts",
+  "keywords": ["amp"],
   "repository": {
     "type": "git",
-    "url": "https://github.com/ampproject/amphtml/tree/master/extensions/amp-viewer-integration/0.1/messaging/"
-  }
+    "url": "git+https://github.com/ampproject/amp-toolbox.git"
```

Comment:
Why repo change here?

Reply:
The docs say this *must* be an actual repo (i.e. URL you can `git clone`) and not a subfolder. That said, I added the `directory` property which should be used for our use-case (subfolder in large repo).

Reply:
Then it should be `https://github.com/ampproject/amphtml.git`, no?

Reply:
Argh, you're right, I overzealously copy-pasted and didn't even notice what you were talking about. Fixed.